docs/user-interface/controls/graphicsview.md

```diff
@@ -0,0 +1,96 @@
+---
+title: ".NET MAUI GraphicsView"
+description: "The .NET MAUI GraphicsView is a graphics canvas on which 2D graphics can be drawn using the cross-platform Microsoft.Maui.Graphics library."
+ms.date: 07/28/2021
+---
+
+# .NET MAUI GraphicsView
+
+<!-- Sample link goes here -->
+
+The .NET Multi-platform App UI (.NET MAUI) `GraphicsView` is a graphics canvas on which 2D graphics can be drawn using the `Microsoft.Maui.Graphics` library. For more information about `Microsoft.Maui.Graphics`, see [.NET MAUI Graphics](~/user-interface/graphics/index.md).
```

Comment:
Do we need to call out the library? Is it just included in the workload? I don't believe any of the winforms/wpf content calls out the library where types or concepts are held as it's just assumed the workload references these libraries for you, always.

Reply:
Maui.Graphics is both in the workload, and a library (available to use in .NET for iOS apps, . UWP apps, WPF apps etc.). Hence at the time of writing I called it out as a library. However my thinking is now this: as the docs are about MAUI, where you don't need to know that it's also a library, I should drop the library references. I could also add a note to the index page saying it's also available for single platform apps as a lib available from NuGet.